aksel.nav.no/website/components/sanity-modules/accordion/Accordion.tsx

```diff
@@ -8,15 +9,40 @@ type AccordionProps = {
 };
```

Comment:
If `node` can be undefined, shouldn't we type it like that?

```
type AccordionProps = {
  node?: AccordionT;
};
```

Reply:
Makes sense, fixed